### PR TITLE
release/front-end

### PR DESCRIPTION
**Pull request by diferpaho**

### CODE REVIEW

**Comment by diferpaho** [comment]
Apply the pull request without removing the branch, the front-end launch branch will be the base for the new branches dedicated to the interface

File: main/templates/consult_report.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+{% block title %}
+    Consultar reporte-Carpa
+{% endblock title %}
+
+
+
+{% block content %}
+
+        <div class="row text-center my-1 mx-lg-5">
+            <h1 class="mt-md-3">Consultar reporte</h1>
```
Comment (jmencisom) [inline main/templates/consult_report.html:12]: Please consider to rename the template:

consult_report.html --> query_report.html

Thanks.

File: main/views.py [hunk 2]
```diff
@@ -2,6 +2,27 @@
 from django.shortcuts import render
 
 
+
```
Comment (danialeresbar) [inline main/views.py:5]: Please two blank spaces between imports and code lines 😉 Thanks

**Review by danialeresbar — COMMENTED** [review]
Please create the URL patterns for each **app** in their respective file 'urls.py'. There is an example [here](https://www.fullstackpython.com/django-urls-path-examples.html), but you should check the official [docs](https://docs.djangoproject.com/en/3.1/ref/urls/).

Your progress:

- [x] Create the URL patterns for **main** application in their respective **urls.py** file.
- [ ] Create the views in the **views.py** file and add them as URL patterns in **urls.py**
- [ ] Include application paths in global project settings

File: main/templates/consult_report.html (hunk 1 above)
Comment (diferpaho) [reply main/templates/consult_report.html]: the change has already been made

File: main/views.py (hunk 2 above)
Comment (diferpaho) [reply main/views.py:5]: ok

**Comment by diferpaho** [comment]
> Please create the URL patterns for each **app** in their respective file 'urls.py'. There is an example [here](https://www.fullstackpython.com/django-urls-path-examples.html), but you should check the official [docs](https://docs.djangoproject.com/en/3.1/ref/urls/).
> 
> Your progress:
> 
> * [x]  Create the URL patterns for **main** application in their respective **urls.py** file.
> * [x]  Create the views in the **views.py** file and add them as URL patterns in **urls.py**
> * [x]  Include application paths in global project settings

I already made the proposed changes, please upload the changes to production, thank you.

**Comment by jmencisom** [comment]
@danialeresbar feel free to accelerate the review of the pull request :)

**Review by danialeresbar — APPROVED** [review]

**Pull request merged**